### ISSUE TITLE
note probability inconsistency

### ISSUE THREAD

**Issue by thijz** [hydrogen-music/hydrogen#796]
Hydrogen version *          : 0d3bb860696c7149cd0eb9f722a9c13751a359c7
Operating system + version  : ubuntu 19.10
Audio driver + version      : any

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
note probability (great feature BTW !!!) is available but it's not :-)

you can dial in a probability value per note : 
![image](https://user-images.githubusercontent.com/1384210/73261473-6f05b400-41cc-11ea-85ee-db414c297c90.png)

however, there is no 'master' dial for probability in the mixer

told you : it's there, but it's not  :-D

**Comment by mauser** [comment]
Hi Thijz,
 
just to get this issue right: You mean that the per-note probability works for you, but you need a general, master probability fader which would affect all notes?

How would the master probability and the per-note probability work together? Multiply both (if set?) ?

**Comment by thijz** [comment]
Hi Sebastian,
This is something that is already implemented on a test branch by @elpescado 
(see https://github.com/hydrogen-music/hydrogen/pull/525#issuecomment-427988004)

i assumed that the master section had to be there to make proper use of the probability, but i guess you could use it without
adding the master section does however greatly improve the usability of this feature

**Comment by elpescado** [comment]
Hi,

Sorry for late reply. I've implemented this feature in two parts. Part one consisted of probability bars in pattern editor, and that part has already been merged and is scheduled for 1.0.0 release. [Part 2.0](https://github.com/hydrogen-music/hydrogen/pull/525) builds upon that wort and expands it further by adding two "master" controls that adjust how random notes are played. I consider that more or less complete (sans documentation), but it hasn't been merged yet. As much as I'd like to see it merged, I don't feel confident that it should be part of 1.0.0 release.
